packages/playwright/example.js

```diff
@@ -1,12 +1,17 @@
-const AxeBuilder = require('@axe-core/playwright').default;
+const { AxeBuilder } = require('@axe-core/playwright');
 const playwright = require('playwright');
 
 (async () => {
   const browser = await playwright.chromium.launch();
```

Comment:
As the other examples are in headless we should also add headless here too: 

```suggestion
  const browser = await playwright.chromium.launch({ headless: true });
```